docker-compose.dev.yaml

```diff
@@ -16,3 +16,4 @@ services:
       - SYSTEM_APPS_REPO
       - PRODUCTION=0
       - LOG_LEVEL=DEBUG
+      - USE_PIXLET_SCHEMA=0
```

Comment:
Change the setting here, it doesn‘t pass through a value from .env, I‘ll change that later